CONTRIBUTING.md

```diff
@@ -79,6 +79,7 @@ There are various files that auto-generated across the ICU4X repository.  Here a
 need to run in order to recreate them.  These files may be run in more comprehensive tests such as those included in `cargo make ci-job-test` or `cargo make ci-all`.
 
 - `cargo make testdata` - regenerates all test data in the `provider/testdata` directory.
+- `cargo make bakeddata experimental` - regenerates baked data in the `provider/data/experimental` directory.
```

Comment:
I couldn't find a note of this anywhere. It's definitely my inexperience with Rust and tracking down where commands come from. But felt it might be helpful to have it in here!